### ISSUE TITLE
onMouseMove: dataRunner is not triggered if naturalEvent is disabled

### ISSUE THREAD

**Issue by PacoDu** [nasa-gibs/worldview#616]
It took me some time to figure why the cursor was not displayed on the legend despite the fact that I created my own color map in the GIBS format. 

It seems that the condition is wrong in onMouseMove function:
 https://github.com/nasa-gibs/worldview/blob/6cbe782049c69571305b8402e41d949eee4bffd5/web/js/map/wv.map.ui.js#L878-L882

I think that the condition should be:
``` javascript
      if ((typeof models.naturalEvents != 'undefined' &&
            models.naturalEvents.active) ||
          (typeof models.data != 'undefined' && models.data.active)) {
        return;
      }
```

**Comment by localjo** [comment]
Thanks for looking into this. I'm going to try to reproduce it. My understanding of the issue you're describing is that when the naturalEvents feature is disabled in Worldview, the dataRunner does not show up on the legend. Is that correct?

**Comment by PacoDu** [comment]
Yes that's correct. The function return before this line: https://github.com/nasa-gibs/worldview/blob/6cbe782049c69571305b8402e41d949eee4bffd5/web/js/map/wv.map.ui.js#L888

Which is not a normal behavior. I just made a pull request that fix this issue.

**Comment by localjo** [comment]
Thanks! 🙂 I'll take a look.

**Comment by PacoDu** [comment]
Let me know if you have any trouble with the fix ;) I'll be happy to modify it

**Comment by localjo** [comment]
@PacoDu we merged your patch into `master` on Friday and released it in version 1.11.0 of Worldview. Thanks again for your contribution!
